sql/core/src/test/scala/org/apache/spark/sql/execution/command/DescribeTableSuiteBase.scala

```diff
@@ -178,4 +178,43 @@ trait DescribeTableSuiteBase extends QueryTest with DDLCommandTestUtils {
       assert(errMsg === "DESC TABLE COLUMN does not support nested column: col.x.")
     }
   }
+
+  test("describe a clustered table") {
+    withNamespaceAndTable("ns", "tbl") { tbl =>
+      sql(s"CREATE TABLE $tbl (col1 STRING, col2 INT) $defaultUsing CLUSTER BY (col1, col2)")
+      val descriptionDf = sql(s"DESC $tbl")
+      assert(descriptionDf.schema.map(field => (field.name, field.dataType)) === Seq(
+        ("col_name", StringType),
+        ("data_type", StringType),
+        ("comment", StringType)))
+      QueryTest.checkAnswer(
+        descriptionDf,
+        Seq(
+          Row("col1", "string", null),
+          Row("col2", "int", null),
+          Row("# Clustering Information", "", ""),
+          Row("# col_name", "data_type", "comment"),
+          Row("col1", "string", null),
+          Row("col2", "int", null)))
+    }
+  }
+
+  test("describe a clustered table with comments on clustering columns") {
+    withNamespaceAndTable("ns", "tbl") { tbl =>
+      sql(s"CREATE TABLE $tbl (col1 STRING) $defaultUsing CLUSTER BY (col1)")
+      sql(s"ALTER TABLE $tbl ALTER COLUMN col1 COMMENT 'this is comment';")
```

Comment:
since we only want to test DESCRIBE TABLE, not ATLER COLUMN, shall we specify the column comment in CREATE TABLE? then we don't need to exclude this test case in the hive test.

Reply:
done. also added a nested column case.